Lighthouse Camera/UI/Main/TTSHelper.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  TTSHelper.swift
```

Comment:
Wait, a TTSHelper and FTE in the same patch? Did you merge two patches accidentally?